runtime/browser/xwalk_content_browser_client.cc

```diff
@@ -82,6 +82,13 @@ namespace {
 // The application-wide singleton of ContentBrowserClient impl.
 XWalkContentBrowserClient* g_browser_client = nullptr;
 
+void CallbackPermisisonStatusWrapper(
```

Comment:
Looks like that typo is also present in the upstream code :) But I think we can change it in our code and then I can submit a patch to fix it upstream as well.